src/lib/evil/evil_unistd.h

```diff
@@ -28,6 +28,11 @@ EVIL_API int execvp(const char *file, char *const argv[]);
 #define W_OK    2       /* Test for write permission.  */
 #define X_OK    0       /* execute permission, originally '1', just a bypass here*/
 #define F_OK    0       /* Test for existence.  */
+
+#define STDIN_FILENO 0
+#define STDOUT_FILENO 1
+#define STDERR_FILENO 2
```

Comment:
As from here: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/fileno?redirectedfrom=MSDN&view=vs-2019

```suggestion
#define STDIN_FILENO __fileno(stdin)
#define STDOUT_FILENO __fileno(stdout)
#define STDERR_FILENO __fileno(stderr)
```